themes/bootstrap5/templates/primo/citation_results.phtml

```diff
@@ -38,12 +38,5 @@
   );
 
   // Override breadcrumbs:
-  $titleLink = $this->translate(
-      $this->pageTitleLink,
-      [
-          '%%url%%' => $this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver)),
-          '%%title%%' => $this->escapeHtml($this->driver->getBreadcrumb()),
-      ]
-  );
-
-  $this->layout()->breadcrumbs = '<li class="breadcrumb-item active" aria-current="page">' . $titleLink . '</li>';
+  $this->breadcrumbs()->set($this->driver->getBreadcrumb(), $this->recordLinker()->getUrl($this->driver))
+    ->add($title, active: true);
```

Comment:
This results in a somewhat redundant breadcrumb trail:

[Title of Book] / Results Cited By "[Title of Book]"

...but it feels like a more appropriate trail than the previous mixed-format HTML, which doesn't fit the new breadcrumb trail model.

Reply:
Makes sense to me. It's hard to avoid the redundancy without making the message vague and difficult to understand.